userguide/tutorials/metrics-newrelic.adoc

```diff
@@ -82,38 +82,55 @@ installing the *New Relic Agent* in the system producing the data.
 
 image::metrics-newrelic-installation.png[align=center]
 
-5. In the box containing the install command, click *Copy Command*. The command will be copied to your clipboard.
+[start=5]
+. In the box containing the install command, click *Copy Command*. The command will be copied to your clipboard.
 
 [NOTE]
 This command includes the API key assigned to you by
 New Relic. This authorizes the agent to do its job. This is your *user* API Key, and should not be confused with the key you will use in the next step.
 
 To install the agent: 
 
-1. Login to your EC2 instance. 
-2. Paste the saved install command to the Ubuntu command line. 
-3. Press Enter. The installation should begin.
+[start=6]
+.. Login to your EC2 instance. 
+.. Paste the saved install command to the Ubuntu command line. 
+.. Press Enter. The installation should begin.
 
-You will be presented with a list of options to install, as shown below. An additional option, Infrastructure Agent, is always installed. Press Enter to select all of the options.
+You will be presented with a list of options to install, as shown below. An additional option, Infrastructure Agent, is always installed.
+
+[start=7]
+. Press Enter to select all of the options.
 
 image::metrics-newrelic-options.png[align=center]
 
 
-You will be presented with several questions including default answers. Press Enter after each question to select the defaults.
+You will be presented with several questions including default answers.
+
+[start=8]
+. Press Enter after each question to select the defaults.
+
+Finally, you will be asked for the MySQL password.
 
-Finally, you will be asked for the MySQL password. Type `root`.
+[start=9]
+. Type `root`.
 
 If all goes well, each installation should report `success`.
 
 
-If you are using the Multi-Tier installation, you will have multiple EC2
+[NOTE]
+*Note:* If you are using the multi-tier installation, you will have multiple EC2
 instances. The agent should be installed separately for each one.
 
-To test the installation, return to the main page. Select *Browse Data* from the top menu, then select *Metrics* from the dropdown.
+== Test the Installation
+
+. Return to the main page. 
+. Select *Browse Data* from the top menu. 
+. Select *Metrics* from the dropdown.
+
```

Comment:
It's a bit unclear which "main page" is being referred to here. Is this the main page on the `New Relic` website?

Reply:
Yes. Is there a clearer way to write this?

Reply:
clarified